src/main/index.ts

```diff
@@ -271,30 +271,57 @@ app.on('web-contents-created', (_ev, contents) => {
     contents.session.storagePath &&
     contents.session.storagePath.indexOf('webxdc_') !== -1
   if (is_webxdc) {
-    contents.on('will-navigate', (e, navigationUrl) => {
+    const webxdc_open_url = (url: string) => {
+      if (url.startsWith('mailto:') || url.startsWith('openpgp4fpr:')) {
+        // handle mailto in dc
+        open_url(url)
+        mainWindow.window?.show()
+      }
+    }
+
+    contents.on('will-navigate', (ev, navigationUrl) => {
       if (navigationUrl.startsWith('webxdc://')) {
         // allow internal webxdc nav
         return
       } else if (navigationUrl.startsWith('mailto:')) {
         // handle mailto in dc
-        e.preventDefault()
-        open_url(navigationUrl)
-        mainWindow.window?.show()
+        ev.preventDefault()
+        webxdc_open_url(navigationUrl)
```

Comment:
Mini nitpick, do we have some style guides around letter casing? In JS world I'd expect method names to be camelCase but also I'm open to follow whatever style is preferred here

Reply:
I changed the casing for `webxdc_open_url`, we can change the rest of the projects var name casings in a different pr.